src/UsdPlusRedeemer.sol

```diff
@@ -159,6 +159,10 @@ contract UsdPlusRedeemer is IUsdPlusRedeemer, UUPSUpgradeable, AccessControlDefa
             ticket = $._nextTicket++;
         }
 
+        if (owner != address(this) && msg.sender != owner) {
```

Comment:
Remove the `owner == address(this)` logic here and below. Set `owner: owner`. This was a holdover from staked USD+.